phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/RepositoriesPanel.java

```diff
@@ -127,6 +127,7 @@ private void populateRepositoryGrid() {
         this.addButton.setText(tr("Add"));
         this.addButton.setOnAction((ActionEvent event) -> {
             AddRepositoryDialog dialog = new AddRepositoryDialog();
+            dialog.initOwner(this.getParent().getScene().getWindow());
```

Comment:
Just asking:
What does this line do?
Does it add the stylesheet from the parent to the dialog?